src/modules/trades.ts

```diff
@@ -15,6 +15,7 @@ export namespace trades {
         let buyAmount = event.params.buyAmount
         let txGasPrice = event.transaction.gasPrice
         let txGasLimit = event.transaction.gasLimit
+        let feeAmount = event.params.feeAmount
```

Comment:
I see you persist the sellAmount/buyAmount. Probably for another PR, but I would suggest you to accumulate in the `Token` entity also these volumes.

Reply:
Already created an issue for this. Implemented on my local but need to ask some questions.

Reply:
Okay, ask us in the issue or chat if u want